views/post-view/src/components/Header/index.tsx

```
@@ -1,68 +1,61 @@
               .filter((o) => o.value !== PostOption.Text)
               .map((option) => {
                 return (
-                  <j-button
-                    size="lg"
-                    square
-                    onClick={() => handlePostClick(option.value)}
-                    value={PostOption.Text}
-                    variant="ghost"
-                  >
-                    <j-icon slot="start" size="md" name={option.icon}></j-icon>
+                  <j-button size="lg" square onClick={() => handlePostClick(option.value)} variant="ghost">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)